browser/ui/styles.css

```
@@ -41,110 +39,237 @@
-  text-transform: lowercase;
+[data-iii-ui="browser"] .br-ui-banner.warn {
+  background: var(--color-warn-muted);
 }
-[data-iii-ui="browser"] .br-ui-page-sub {
-  margin: 2px 0 0;
-  font-size: 12px;
+[data-iii-ui="browser"] .br-ui-banner.alert {
+  background: var(--color-alert-muted);
+}
+[data-iii-ui="browser"] .br-ui-banner > span {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)